python/mpt/tests/src/mpt/test_utils.py

```diff
@@ -112,6 +118,30 @@ def leaf_node_could_be_invalid_strategy(draw):
             return path, LeafNode(rest_of_key=rest_of_key, value=value)
 
 
+@st.composite
+def extension_node_could_be_invalid_strategy(draw):
```

Comment:
invalid subnode could be:
None
Empty List (empty embedded node)
Empty Bytes (i.e. empty hash32 digest)